lib/cluster.js

```diff
@@ -155,7 +155,7 @@ RoundRobinHandle.prototype.add = function(worker, send) {
   if (this.server === null) return done();
   // Still busy binding.
   this.server.once('listening', done);
-  this.server.once('error', function(err) {
+  this.server.once('error', function errorCodeTranslation(err) {
```

Comment:
name does not describe purpose. it does do some translation, but then it sends, so sending the error seems to be the main purpose. I don't personally support this renaming, and don't think removing anonymous functions has any improvement on the comprehensibility of stack traces, and certainly does not when the anonymous functions are given names not indicative of what they do.